### DISCUSSION TITLE
Permissões GitHub

### DISCUSSION THREAD

**Discussion by aldelgado2** [frontendbr/forum#2387] [Perguntas e Respostas]
Prezados,

Estou iniciando no Git / GitHub e estou com uma dúvida, quem puder me ajudar eu agradeço.

Tenho um repositório privado, consigo compartilhar ele via link para terceiros sem que a pessoa faça qualquer tipo de alteração ou cópia dos dados?

Caso seja possível, como proceder?

Obrigado.

Alexandre

**Comment by igorlvicente** [comment]
[Aqui ](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository) você aprende como convidar alguém (colaborador) para seu repositório e [aqui](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/permission-levels-for-a-personal-account-repository#collaborator-access-for-a-repository-owned-by-a-personal-account) você aprende sobre as permissões de um colaborador. Nesse último link temos o seguinte aviso:
> Note: In a private repository, repository owners can only grant write access to collaborators. Collaborators can't have read-only access to repositories owned by a personal account.

Então a resposta é **não**. **Você não consegue compartilhar ele via link** e se você adiciona uma pessoa como colaboradora, **ela poderá fazer alterações no repositório também**.

O motivo disso provavelmente é que essa funcionalidade é paga nos planos empresariais.

**Comment by aldelgado2** [comment]
Bom dia Igor,

Vou estudar o material que me enviou, muito obrigado.

Abs,

Alexandre

Em sex., 6 de jan. de 2023 às 10:25, Igor L. Vicente <
***@***.***> escreveu:

**Comment by igorlvicente** [comment]
Se não estou errado, essa funcionalidade está disponível de graça no GitLab. Dê uma olhada lá.
Link para a documentação: https://docs.gitlab.com/ee/user/project/members/#add-users-to-a-project

**Comment by aldelgado2** [comment]
Vou verificar.

Obrigado.

Abs,

Alexandre

Em sex., 6 de jan. de 2023 às 11:51, Igor L. Vicente <
***@***.***> escreveu:
